From ca412d4f25fc986aa4f5d6532de9147d9a5bb4c0 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 3 May 2021 19:57:16 +0100
Subject: [PATCH 1/2] gnu: cl-environments: Update to v0.4

* gnu/package/lisp-xyz.scm (sbcl-cl-environments)
  [version]: use annotated tag instead of direct commit, there is
  upstream releases available now
---
 gnu/packages/lisp-xyz.scm | 59 +++++++++++++++++++--------------------
 1 file changed, 29 insertions(+), 30 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 57b4abf067..b2de6965e0 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -13135,36 +13135,35 @@ collecting values easier.")
   (sbcl-package->ecl-package sbcl-collectors))
 
 (define-public sbcl-cl-environments
-  (let ((commit "0b22154c5afefef23d1eba9a4fae11d73580ef41")) ; No version in 2 years.
-    (package
-      (name "sbcl-cl-environments")
-      (version (git-version "0.2.3" "1" commit))
-      (source
-       (origin
-         (method git-fetch)
-         (uri (git-reference
-               (url "https://github.com/alex-gutev/cl-environments")
-               (commit commit)))
-         (file-name (git-file-name name version))
-         (sha256
-          (base32
-           "18r3wfarr7lgn78m6c66r0r9aazirv07gy7xgvqkl9pmrz1bc47m"))))
-      (build-system asdf-build-system/sbcl)
-      (inputs
-       `(("alexandria" ,sbcl-alexandria)
-         ("anaphora" ,sbcl-anaphora)
-         ("collectors" ,sbcl-collectors)
-         ("optima" ,sbcl-optima)))
-      (native-inputs
-       `(("prove" ,sbcl-prove)))
-      (home-page "https://github.com/alex-gutev/cl-environments")
-      (synopsis "Implements the Common Lisp standard environment access API")
-      (description "This library provides a uniform API, as specified in Common
-Lisp the Language 2, for accessing information about variable and function
-bindings from implementation-defined lexical environment objects.  All major
-Common Lisp implementations are supported, even those which don't support the
-CLTL2 environment access API.")
-      (license license:expat))))
+  (package
+    (name "sbcl-cl-environments")
+    (version "0.4")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/alex-gutev/cl-environments")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "10jxj043d2dw5vc0i0lz0lsa4qszn8him5is8jdhl4nsyfcazmky"))))
+    (build-system asdf-build-system/sbcl)
+    (inputs
+     `(("alexandria" ,sbcl-alexandria)
+       ("anaphora" ,sbcl-anaphora)
+       ("collectors" ,sbcl-collectors)
+       ("optima" ,sbcl-optima)))
+    (native-inputs
+     `(("prove" ,sbcl-prove)))
+    (home-page "https://github.com/alex-gutev/cl-environments")
+    (synopsis "Implements the Common Lisp standard environment access API")
+    (description
+     "This library provides a uniform API, as specified in Common Lisp the
+Language 2, for accessing information about variable and function bindings
+from implementation-defined lexical environment objects.  All major Common
+Lisp implementations are supported, even those which don't support the CLTL2
+environment access API.")
+    (license license:expat)))
 
 (define-public cl-environments
   (sbcl-package->cl-source-package sbcl-cl-environments))
-- 
2.31.1

